File: Wms.Api/Dto/CartonSize/CartonSize.cs
 using Wms.Api.Entities; 

namespace Wms.Api.Dto.CartonSize
{
    public class CartonSizeDetailsDto : CreatedChangedEntity
    {
        public Guid Id { get; set; }
        public required string Name { get; set; }
    }

    public class CartonSizeCreateUpdateDto
    {
        public required string Name { get; set; }
    }

    public class CartonSizeSearchDto : PagedRequestAbstractDto
    {
        public required string Search { get; set; }
    }
}

File: Frontend/src/interfaces/v12/cartonSize/cartonSize.ts
import { CreatedChangedEntity } from "interfaces/v12/CreatedChangedEntity";
import { guid } from "types/guid";
import { PagedRequestAbstractDto } from "../pagedRequest/pagedRequestAbstractDto";

export interface CartonSizeDetailsDto extends CreatedChangedEntity {
  id: guid;
  name: string;
}

export interface CartonSizeCreateUpdateDto {
  name: string;
}

export interface CartonSizeSearchDto extends PagedRequestAbstractDto {
  search: string;
}
